cmd/vic-machine/create/create.go

```diff
@@ -308,8 +308,8 @@ func (c *Create) processParams() error {
 		return err
 	}
 
-	if c.ImageDatastoreName == "" {
-		return cli.NewExitError("--image-datastore Image datastore name must be specified", 1)
+	if c.ImageDatastorePath == "" {
+		return cli.NewExitError("--image-datastore Image datastore path must be specified", 1)
```

Comment:
Let's supply a message that avoids the need to refer to the documentaiton for format.
`--image-store must be specified, e..g datastore1 or datastore1/store`